snap/snapcraft.yaml

```diff
@@ -25,37 +25,22 @@ apps:
       - dbus-daemon
     common-id: com.github.unrud.VideoDownloader
     desktop: usr/share/applications/com.github.unrud.VideoDownloader.desktop
+    environment:
+      PATH: $SNAP_DESKTOP_RUNTIME/usr/bin:$PATH
+      PYTHONPATH: $PYTHONPATH:$SNAP_DESKTOP_RUNTIME/usr/lib/python3.6/site-packages:$SNAP/usr/lib/python3.6/site-packages:$SNAP/lib/python3.6/site-packages
```

Comment:
Apparently this does the trick:

```
PYTHONPATH: "${SNAP}/gnome-platform/usr/lib/python3/site-packages"
```

It now works with `/snap/video-downloader/current/usr/bin/python3` and `/snap/video-downloader/current/gnome-platform/usr/bin/python3`. That's strange.